src/uct/api/uct.h

```diff
@@ -1420,6 +1423,13 @@ struct uct_ep_params {
      * @ref UCT_IFACE_FLAG_CONNECT_TO_SOCKADDR capability.
      */
     const ucs_sock_addr_t             *local_sockaddr;
+
+    /**
+     * On supported transports, an additional uct_ep is created
+     * for higher priority messages, which can be handled differently in
+     * the underlying transport.
```

Comment:
This documents the usage and not the parameter
It should be something like: 
`Priority level of operations on this endpoint, supported values are 0..@ref uct_iface_attr_v2_t::max_priority-1. if not specified, the priority is considered to be 0, which is the lowest priority.`